broker/applications/osb-broker/src/api-controllers/ServiceBrokerApiController.js

```diff
@@ -631,6 +631,11 @@ class ServiceBrokerApiController extends FabrikBaseController {
             }
             _.set(body,'service_id',_.get(resource, 'spec.serviceId'));
             _.set(body,'plan_id',_.get(resource, 'spec.planId'));
+            
+            if(_.has(resource, 'spec.metadata')) {
+              _.set(body, 'matadata', _.get(resource, 'spec.metadata'));
```

Comment:
```suggestion
              _.set(body, 'metadata', _.get(resource, 'spec.metadata'));
```
Minor typo.

Reply:
Thanks @anoopjb for noting this. Will add a UT for this for better visibility.

Reply:
Added UT.